docs/[dir]/dynamic.md.js

```diff
@@ -0,0 +1,23 @@
+import {parseArgs} from "node:util";
+
+const {
+  values: {dir}
+} = parseArgs({
+  options: {dir: {type: "string"}}
+});
+
+console.log(`# Hello ${dir}
```

Comment:
Are we adopting `console.log` (I thought we wanted to use `process.stdout.write`). I don't have a strong preference, but somehow a lot of users struggle with this because they're so used to `console.log` for debugging.